### ISSUE TITLE
RuntimeWarning: invalid value encountered in double_scalars when computing weights

### ISSUE THREAD

**Issue by jimbudarz** [TeamHG-Memex/eli5#369]
Hi, I'm experiencing an error in `eli5.show_weights()`, I believe it's where the colors are being computed. I've followed the simplest tutorial (https://eli5.readthedocs.io/en/latest/blackbox/permutation_importance.html) and it's not working with my dataset and model.

I'm using eli5 version 0.10.1

Here is the relevant code:
`def permutation_importance(X_test, y_test, clf):`
`import eli5`
`from eli5.sklearn import PermutationImportance`
`perm = PermutationImportance(clf).fit(X_test, y_test)`
`eli5.show_weights(perm)`

`permutation_importance(X_test_dummified,y_test,rf_clf_imbalanced)`

Here is the entire error log:
> ~/anaconda3/envs/aif360/lib/python3.7/site-packages/eli5/formatters/html.py:235: RuntimeWarning: invalid value encountered in double_scalars
>   rel_weight = (abs(weight) / weight_range) ** 0.7

The error is probably with `weight` or `weight_range`, but the error log does not specify where these are generated.

Can anyone help me trace this error back? Thanks!

**Comment by jimbudarz** [comment]
Additional information: I've now found that `eli5.explain_weights(perm)` results in all weights of zero, so the problem is originating somewhere else.

**Comment by XinQi7788** [comment]
I encountered the same issue. Can anyone help on this issue? 

Thanks a lot!
